config/ols-redis.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: redis-insights-route
+  namespace: ols
+spec:
+  to:
+    kind: Service
+    name: redis-stack
+  port:
+    targetPort: 8001
+  wildcardPolicy: None
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: redis-stack
+  namespace: ols
+spec:
+  selector:
+    app: redis
+  ports:
+    - name: redis-server
+      protocol: TCP
+      port: 6379
+      targetPort: 6379
+    - name: redis-insights
+      protocol: TCP
+      port: 8001
+      targetPort: 8001
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: redis-stack
+  namespace: ols
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: redis
+  template:
+    metadata:
+      labels:
+        app: redis
+    spec:
+      containers:
+      - name: redis
+        image: ${OLS_REDIS_IMAGE}
```

Comment:
w/ the reliance on "envsubst" to populate these things, we need to provide defaults somewhere.

Reply:
kustomize doesn't provide a way to achieve this. For now we have to rely on "evnsubst". And in future we can make our python code to deploy these resources by populating those values in the runtime.